File: cpp/bc2373.cpp
#include <iostream>

using namespace std;

int main()
{
    int count, nCo, nLa, tCo = 0;

    cin >> count;

    for(int i = 0; i < count; i++)
    {
        cin >> nLa >> nCo;
        if(nCo < nLa) tCo += nCo;
    }
    cout << tCo << endl;
}

File: python/bc2373.py
tCo = 0
count = int(input())

for i in range(count):
    cVal = input().split(" ")

    nLa = int(cVal[0])
    nCo = int(cVal[1])

    if nCo < nLa: tCo += nCo

print(tCo)